geoportailv3/static/js/layerinfo/layerinfoservice.js

```diff
@@ -14,32 +14,112 @@ goog.require('ngeo.CreatePopup');
 app.ShowLayerinfo;
 
 
+
 /**
  * @param {angular.$http} $http Angular $http service
  * @param {angular.$sce} $sce Angular $sce service
+ * @param {angular.Scope} $rootScope The rootScope provider.
+ * @param {angularGettext.Catalog} gettextCatalog Gettext catalog.
  * @param {ngeo.CreatePopup} ngeoCreatePopup Ngeo popup factory service
- * @return {app.ShowLayerinfo} The show layer info function.
+ * @constructor
```

Comment:
If we use a `@constructor` (and `module.service` instead of `module.factory`) then the function name should start with a capital letter and the should not be suffixed with `Factory`.

In that case specific case it's not a constructor that we want, it's a simple function that returns a function. So what you want to do is just to change `app.module.service` to `app.module.factory` for the declaration of the `appShowLayerInfo` service.